hooks/96-auto-manage-uefi-entries.install

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+check_if_uefi_entry_exists () {
+	for entry in $(efibootmgr); do
+
+		if [[ "$entry" == *"($efi_path"* ]] && [[ "$entry" == *"$partition_uuid"* ]]; then
+			
+			return 1
+		fi
+			
+	done
+	return 0
+}
+
+add_uefi_entry () (
+	for efi in $partition_efis; do
+		
+		# Prepare path which will be inserted into efibootmgr
+		efi_path=$(echo "${efi//$partition_mount}" | sed 's/\//\\/g')
+
+		# Add entry if it doesn't exist
+		if check_if_uefi_entry_exists; then
+
+			echo "Adding UEFI entry for $efi_path found on $partition..."
+			efibootmgr --create --disk /dev/$partition --label "Gentoo $partition" --loader "$efi_path" -u $kenrel_commands &>/dev/null
+		
+		else
+		
+			echo "UEFI Entry already exists for $efi_path on $partition !"	
+		
+		fi
+	done
+)
+
+main () {
+	efi_parttype="c12a7328-f81f-11d2-ba4b-00a0c93ec93b"
+	kernel_commands='crypt_root=UUID=dcb0cc6f-464c-4e98-b91c-e59e3655d161 root=/dev/mapper/gentoo-root rootfstype=ext4 resume=/dev/mapper/gentoo-swap dolvm quiet'
+	mounted_esp=$(lsblk -lo NAME,MOUNTPOINTS,PARTTYPE | grep  "$efi_parttype" | grep "/" | cut -d' ' -f1)
+	
+	for partition in $mounted_esp; do
```

Comment:
Are there actually situations where one system would have multiple ESPs mounted?

Reply:
We have RAID 1 of 2-3 disks, with EFI partition on each of the (outside of raid). In case of device failure of 1 disk, we will be able to boot EFI file from the second or third disk

Reply:
But do the other hooks follow the same logic?

Reply:
> But do the other hooks follow the same logic?

Other hooks determine the location of the ESP from `KERNEL_INSTALL_BOOT_ROOT`, this detection is done by `kernel-install`.

Reply:
Then IMHO _this PR_ should introduce a new logic.
cc @Biosias